src/components/application_manager/src/commands/button_notification_to_mobile.cc

```diff
@@ -0,0 +1,168 @@
+#include "application_manager/commands/button_notification_to_mobile.h"
+namespace application_manager {
+
+namespace commands {
+namespace app_mngr = application_manager;
+
+CREATE_LOGGERPTR_GLOBAL(logger_, "ButtonNotificationToMobile")
+
+ButtonNotificationToMobile::ButtonNotificationToMobile(
+    const app_mngr::commands::MessageSharedPtr& message,
+    app_mngr::ApplicationManager& application_manager,
+    app_mngr::rpc_service::RPCService& rpc_service,
+    app_mngr::HMICapabilities& hmi_capabilities,
+    policy::PolicyHandlerInterface& policy_handler)
+    : CommandNotificationImpl(message,
+                              application_manager,
+                              rpc_service,
+                              hmi_capabilities,
+                              policy_handler) {}
+
+ButtonNotificationToMobile::~ButtonNotificationToMobile() {}
+
+bool ButtonNotificationToMobile::DoesAppIDExist() const {
+  LOG4CXX_AUTO_TRACE(logger_);
+  using namespace application_manager;
```

Comment:
@mked-luxoft ` using namespace application_manager;` -> ` using application_manager::strings;`? No?

Reply:
@AByzhynar fixed in 704a3e5e302aba1557830edf5e6afedb18c317e3